app/controllers/ctc/questions/confirm_payment_controller.rb

```diff
@@ -11,7 +11,8 @@ def edit
         @ctc_amount = benefits.outstanding_ctc_amount
         @third_stimulus_amount = benefits.outstanding_eip3
         @eitc_amount = benefits.claiming_and_qualified_for_eitc? ? benefits.eitc_amount : nil
-        @total_amount = [@ctc_amount, @third_stimulus_amount, @eitc_amount].compact.sum
+        @fed_income_tax_withholding_amount = current_intake.w2s.pluck(:federal_income_tax_withheld).sum
```

Comment:
We should probably have a test for this in the controller spec where we create multipe w2s for an intake

Reply:
I paired with Whitney and we decided to update this line to use the existing method for calculating the fed_income_tax_withholding_amount. Since there are tests for the the method already we decided that would be sufficient as far as testing this instance variable here. Let me know if you agree with that!